draft-ietf-sedate-datetime-extended.md

```diff
@@ -435,6 +427,49 @@ required to reject or perform some other error handling when
 encountering inconsistent or unrecognized suffix tags marked as
 critical.
 
+## Offset vs. Time Zone Conflicts
+
+Time zone offsets in a timestamp string can conflict with its time zone suffix.
+For example, a calendar application could store an IXDTF string representing a
+far-future meeting in a particular time zone. If that time zone's definition is
+subsequently changed to abolish Daylight Saving Time, IXDTF strings that were
+originally correct may now be conflicting.
+
+If time zone and offset are in conflict an application MUST (if the critical
+flag is used on the time zone hint) or MAY (if the critical flag is not used)
+reject the timestamp or resolve the conflict via user and/or programmer input.
+
+## Use of `Z` Offset Designator with Time Zone Suffix
```

Comment:
I think I would like this text (up to, but not including, the examples) for the "update 3339" section.

Reply:
Sure, that works for me. Do you want me to remove it from here?